includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -17,14 +17,20 @@ public function getData() {
 		global $wgUser;
 
 		$data = [
+			// TODO: restore old logo before 4th October 2016
+			// https://wikia-inc.atlassian.net/browse/XW-1966
 			'logo' => [
 				'header' => [
 					'type' => 'link-image',
 					'href' => $this->getHref( 'fandom-logo' ),
-					'image' => 'wds-company-logo-fandom-powered-by-wikia',
+					'image' => 'wds-company-logo-wikia',
 					'title' => [
-						'type' => 'text',
-						'value' => 'Fandom powered by Wikia'
+						'type' => 'translatable-text',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "translatable-text" 23 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)